client/state/analytics/selectors.js

```diff
@@ -0,0 +1,3 @@
+export function isTracking( state ) {
+	return state.isTracking;
+}
```

Comment:
please note that we are adding all new selectors to the flat global directory `client/state/selectors`